src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -115,16 +115,14 @@ internal static NamespaceOrTypeSymbol ContainingNamespaceOrType(this Symbol symb
             return null;
         }
 
-        internal static Symbol ContainingNonLambdaMember(this Symbol containingMember)
+        internal static Symbol ContainingNonLambdaMember(this Symbol member)
```

Comment:
I can revert. I refactored it to a more readable for as I implemented it for VB.